great_expectations/rule_based_profiler/config/base.py

```diff
@@ -253,6 +257,12 @@ class Meta:
 
     __config_class__ = RuleBasedProfilerConfig
 
+    class_name = fields.String(required=True)
+    module_name = fields.String(
+        required=False,
+        all_none=True,
+        missing="great_expectations.rule_based_profiler",
+    )
```

Comment:
Top level class and module names enable `instantiate_class_from_config` and `test_yaml_config` functionality